### PR TITLE
add identities button

### PR DESCRIPTION
**Pull request by zanderz**

### CODE REVIEW

**Comment by zanderz** [comment]
So this should be done over again after your big identv3 PR it sounds like.

**Comment by chrisnojima** [comment]
Nothing is blocked here just put it in the right place

**Comment by zanderz** [comment]
ok @chrisnojima how about now?

File: shared/__mocks__/feature-flags.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ if (!__STORYBOOK__) {
 }
 
 const ff: FeatureFlags = {
+  addIdentity: true,
```
Comment (chrisnojima) [inline shared/__mocks__/feature-flags.js]: this feature flag should be used for all the generic proofs so it should likely be about that, not just this one button

File: shared/styles/colors.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ const colors = {
   black_50_on_white: 'rgb(127,127,127)',
   black_60: 'rgba(0, 0, 0, 0.60)',
   black_63: 'rgba(0, 0, 0, 0.63)',
+  black_75: 'rgba(0, 0, 0, 0.75)',
```
Comment (chrisnojima) [inline shared/styles/colors.js]: @cecileboucheron is this correct

Comment (adamjspooner) [reply shared/styles/colors.js]: `black` (at 85%) replaced `black_75`.

File: shared/profile/user/index.js [hunk 3]
```diff
@@ -484,6 +505,9 @@ const styles = Styles.styleSheetCreate({
   }),
   innerContainer: {...Styles.globalStyles.fillAbsolute},
   invisible: {opacity: 0},
+  label: {
+    color: Styles.globalColors.black_75,
```
Comment (adamjspooner) [inline shared/profile/user/index.js]: This should be `black`.

File: shared/styles/colors.js (hunk 2 above)
Comment (zanderz) [reply shared/styles/colors.js]: so the zeplin file is not the source of truth? https://app.zeplin.io/project/56b25ffc64fc91773a73aaf8/screen/5bf33ccc2790467ebf9a9bfc

Comment (cecileboucheron) [reply shared/styles/colors.js]: No let's remove that line and use `globalColors.black`. We darkened our blackest black and decided to just call it `black`.

File: shared/__mocks__/feature-flags.js (hunk 1 above)
Comment (zanderz) [reply shared/__mocks__/feature-flags.js]: :white_check_mark:

File: shared/styles/colors.js (hunk 2 above)
Comment (zanderz) [reply shared/styles/colors.js]: :white_check_mark:

File: shared/profile/user/index.js (hunk 3 above)
Comment (zanderz) [reply shared/profile/user/index.js]: :white_check_mark:

File: shared/profile/user/index.js [hunk 4]
```diff
@@ -250,6 +252,17 @@ class BioTeamProofs extends React.PureComponent<BioTeamProofsProps> {
           </Kb.Text>
           <Teams username={this.props.username} />
           <Proofs {...this.props} />
+          {flags.proofProviders && (
+            <Kb.Box2 direction="horizontal" style={styles.addIdentityContainer}>
```
Comment (chrisnojima) [inline shared/profile/user/index.js]: this block should only show up given some other kind of prop. as 1. it should only show if there are additional identities and 2. if should only show on your own profile, so we at least need `props.showOtherIdentities` or something

Comment (zanderz) [reply shared/profile/user/index.js]: :white_check_mark:

**Pull request merged**